<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>588</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe Script</family>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 127);
border-bottom: 1px solid;
	border-bottom-color: #c3c3c3;
border-right: 1px solid;
	border-right-color: #c3c3c3;
border-radius: 2px;



</string>
      </property>
      <property name="text">
       <string>LISTA DE TAREFAS</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="2">
     <widget class="QListWidget" name="listWidget">
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid #c3c3c3;
background-color: rgb(240, 240, 240);
border-radius: 2px</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="gridVisible">
       <bool>true</bool>
      </property>
      <property name="horizontalHeaderFormat">
       <enum>QCalendarWidget::ShortDayNames</enum>
      </property>
      <property name="verticalHeaderFormat">
       <enum>QCalendarWidget::NoVerticalHeader</enum>
      </property>
      <property name="navigationBarVisible">
       <bool>true</bool>
      </property>
      <property name="dateEditEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QWidget" name="widget_img" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>270</width>
        <height>200</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>270</width>
        <height>200</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QTextEdit" name="textEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>270</width>
           <height>200</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Vladimir Script</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">
background-color: transparent;
border: transparent;</string>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Vladimir Script'; font-size:20pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btn_adicionar">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
color: white;
border-right:  1px solid;
	border-right-color: #6a6a6a;
border-bottom: 1px solid;
	border-bottom-color:#6a6a6a;
background-color:#202bff;
border-radius: 1px
}

QPushButton:pressed{
color: black;
border-left: 1px solid;
	border-left-color: #c3c3c3;
border-top: 1px solid;
	border-top-color:#c3c3c3;
border-right: transparent;
border-bottom: transparent;
background-color: rgb(240, 240, 240);
border-radius: 1px
}

</string>
        </property>
        <property name="text">
         <string>Adicionar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_excluir">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
color: white;
border-right:  1px solid;
	border-right-color: #6a6a6a;
border-bottom: 1px solid;
	border-bottom-color:#6a6a6a;
background-color:#202bff;
border-radius: 1px
}

QPushButton:pressed{
color: black;
border-left: 1px solid;
	border-left-color: #c3c3c3;
border-top: 1px solid;
	border-top-color:#c3c3c3;
border-right: transparent;
border-bottom: transparent;
background-color: rgb(240, 240, 240);
border-radius: 1px
}

</string>
        </property>
        <property name="text">
         <string>Excluir</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_marcar">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
color: white;
border-right:  1px solid;
	border-right-color: #6a6a6a;
border-bottom: 1px solid;
	border-bottom-color:#6a6a6a;
background-color:#202bff;
border-radius: 1px
}

QPushButton:pressed{
color: black;
border-left: 1px solid;
	border-left-color: #c3c3c3;
border-top: 1px solid;
	border-top-color:#c3c3c3;
border-right: transparent;
border-bottom: transparent;
background-color: rgb(240, 240, 240);
border-radius: 1px
}

</string>
        </property>
        <property name="text">
         <string>Marcar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_desmarcar">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
color: white;
border-right:  1px solid;
	border-right-color: #6a6a6a;
border-bottom: 1px solid;
	border-bottom-color:#6a6a6a;
background-color:#202bff;
border-radius: 1px
}

QPushButton:pressed{
color: black;
border-left: 1px solid;
	border-left-color: #c3c3c3;
border-top: 1px solid;
	border-top-color:#c3c3c3;
border-right: transparent;
border-bottom: transparent;
background-color: rgb(240, 240, 240);
border-radius: 1px
}

</string>
        </property>
        <property name="text">
         <string>Desmarcar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_limpar">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
color: white;
border-right:  1px solid;
	border-right-color: #6a6a6a;
border-bottom: 1px solid;
	border-bottom-color:#6a6a6a;
background-color:#202bff;
border-radius: 1px
}

QPushButton:pressed{
color: black;
border-left: 1px solid;
	border-left-color: #c3c3c3;
border-top: 1px solid;
	border-top-color:#c3c3c3;
border-right: transparent;
border-bottom: transparent;
background-color: rgb(240, 240, 240);
border-radius: 1px
}

</string>
        </property>
        <property name="text">
         <string>Limpar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 127);
border-bottom: 1px solid;
	border-bottom-color: #c3c3c3;
border-right: 1px solid;
	border-right-color: #c3c3c3;
border-radius: 2px;
</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="styleSheet">
            <string notr="true">background-color: transparent;
border: transparent;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">background-color: transparent;
border: transparent;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_voltar">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
color: black;
border-right:  1px solid;
	border-right-color: #6a6a6a;
border-bottom: 1px solid;
	border-bottom-color:#6a6a6a;
background-color: rgb(240, 240, 240);
border-radius: 1px
}

QPushButton:pressed{
color: black;
border-left: 1px solid;
	border-left-color: #c3c3c3;
border-top: 1px solid;
	border-top-color:#c3c3c3;
border-right: transparent;
border-bottom: transparent;
background-color: rgb(240, 240, 240);
border-radius: 1px
}

</string>
           </property>
           <property name="text">
            <string>voltar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLineEdit" name="lineEdit">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid #c3c3c3;
background-color: rgb(240, 240, 240);
border-radius: 2px</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
